soroban-env-host/Cargo.toml

```diff
@@ -70,7 +70,7 @@ wasmprinter = "0.2.72"
 expect-test = "1.4.1"
 more-asserts = "0.3.1"
 pretty_assertions = "1.4.0"
-backtrace = "0.3.69"
+backtrace = "0.3.75"
```

Comment:
Is this change necessary? Note, that we'll likely not pick this up in core until the next protocol release (unless there is some breaking issue, in which case we'll need an env release).

Reply:
This was to get the CI to pass. cargo-deny was complaining about a deprecated transitive dependency, and bumping the backtrace version made the CI happy.

Reply:
FWIW we don't have to update the crates and we can relax the cargo-deny check if necessary. But this update seems harmless enough to me.
cc @graydon to confirm